include/ck/utility/generic_memory_space_atomic_max.hpp

```diff
@@ -0,0 +1,29 @@
+#pragma once
```

Comment:
Combine this file with ```generic_memory_space_atomic_add.hpp``` into ```include/ck/utility/generic_memory_space_atomic.hpp```

Reply:
Fix in e046063d86da95fff67db1a87323fc8374540c8e